lib/content/utils.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const _ = require('lodash');
+const isUUID = require('validator/lib/isUUID');
+
+
+const check = {
+
+  /**
```

Comment:
Not blocking, but I'd prefer these to be top-level functions that get bundled together in to `check` in the `module.exports`